setup.py

```diff
@@ -16,15 +16,15 @@
 history = open('CHANGES.rst').read()
 
 tests_require = [
-    'check-manifest>=0.25',
-    'coverage>=4.0',
+    'check-manifest>=0.35',
+    'coverage>=4.5.3',
```

Comment:
Based on the [new dependency management](https://invenio.readthedocs.io/en/latest/releases/v3.3.0.html#dependency-management) we should be able to remove coverage, isort, check-manifest and pytest from here. 

I would also try with 
- pydocstyle
- pytest-cov

Reply:
Not really... I just realized `isort`, `coverage` and `check-manifest` are only in `tests_requires`. I am not sure they will get installed here unless they are in `install_requires`. No?

Reply:
Talked with @slint that is missing from pytest-invenio. I'm making a PR now.